app/src/main/res/layout/fragment_homepage.xml

```diff
@@ -391,16 +344,42 @@
 
                 <TextView
                     android:id="@+id/Amp_Pick_up_info"
-                    android:layout_width="205dp"
+                    android:layout_width="235dp"
                     android:layout_height="wrap_content"
                     android:gravity="left"
+                    android:padding="26dp"
                     android:text="Notes in Amp means if the robot scored in the Amp which is needed to amplify the speaker. "
                     android:textColor="@color/white"
                     android:textSize="15sp"
-                    android:padding="26dp"
                     tools:ignore="HardcodedText,RtlHardcoded" />
             </TableRow>
 
+            <TableRow
+                android:layout_width="match_parent"
+                android:layout_height="match_parent">
+
+                <TextView
+                    android:id="@+id/Missed_Notes"
+                    android:layout_width="14dp"
+                    android:layout_height="wrap_content"
+                    android:gravity="center"
+                    android:text="Missed Notes in Speaker or Amp"
+                    android:textColor="@color/white"
+                    android:textSize="30sp"
+                    android:padding="10dp"
+                    tools:ignore="HardcodedText,RtlHardcoded" />
+
+                <TextView
+                    android:id="@+id/Missed_Notes_info"
+                    android:layout_width="259dp"
+                    android:layout_height="151dp"
+                    android:gravity="left"
+                    android:padding="26dp"
+                    android:text="If the robot missed a note to a speaker or amp. Press the + button. "
```

Comment:
Can you add that this is only for notes missed in teleop? there were a couple scouters that thought that missed notes was also for teleop

Reply:
was also for teleop? isnt is for teleop?

Reply:
sorry yeah I mean that it's only for teleop. But since robots also miss during auto there were some scouters that thought that those would also be counted when they shouldn't be

Reply:
done